code/modules/admin/create_poll.dm

```diff
@@ -5,6 +5,34 @@
 	if(!dbcon.IsConnected())
 		src << "<span class='danger'>Failed to establish database connection.</span>"
 		return
+	var/returned = create_poll_function()
+	if(returned)
+		var/DBQuery/query_check_option = dbcon.NewQuery("SELECT id FROM [format_table_name("poll_option")] WHERE pollid = [returned]")
```

Comment:
[You should do the opposite of what this song suggests](https://www.youtube.com/watch?v=_R4qb6_RPUc)